src/frontend/src/handler/explain.rs

```diff
@@ -212,7 +213,10 @@ async fn do_handle_explain(
             ExplainType::Physical => {
                 // if explain trace is on, the plan has been in the rows
                 if !explain_trace && let Ok(plan) = &plan {
-                    blocks.push(plan.explain_to_string());
+                    match explain_format {
+                        ExplainFormat::Text => blocks.push(plan.explain_to_string()),
+                        ExplainFormat::Json => blocks.push(plan.explain_to_json()),
+                    }
```

Comment:
Only physical plan is supported?

Reply:
From test cases it seems they are supported. The code diff looks strange though

Reply:
BTW, how will `explain (trace)` look like?

Reply:
It will ignore the `JSON FORMAT`, and just output text.

Reply:
Hmmm, should we reject it instead?

Reply:
Fixed